### ISSUE TITLE
Accessbility

### ISSUE THREAD

**Issue by SamerElaila** [IsraaMoqbel/experiment#14]
https://github.com/IsraaMoqbel/experiment/blob/2c03dd32871b15f008272f76cc24ae99298f5240/src/views/partials/footer.hbs#L2

This is not semantic html,
footer should be wrapped in `<footer>` tag for accessibility
